sycl/include/sycl/detail/backend_traits_level_zero.hpp

```diff
@@ -135,16 +135,6 @@ template <> struct BackendInput<backend::ext_oneapi_level_zero, queue> {
 
     device Device;
 
-    type()
```

Comment:
Was it intentional removal of this default constructor? 
I am asking because it is not marked as deprecated.

Reply:
Yes,  all default constructor that default-construct the device without passing the device is deprecated.

Reply:
Ok, thank you.